input/ig.xml

```diff
@@ -665,15 +665,15 @@
 				<reference value="Procedure/dialysis"/>
 			</reference>
 			<name value="Procedure - Dialysis"/>
-			<description value="Dialysis &#xa;&#xa;Patient: Mr. Harry Dan"/>
+			<description value="Shows an example *AU Core Procedure*, with reason as a reference to a Condition resource.&#xa;&#xa;Patient: Ronny Lawrence Irvine"/>
```

Comment:
Procedures are all missing information like what the procedure is and other details.